frontend/sw360-portlet/src/main/webapp/html/projects/includes/linkedProjects.jspf

```diff
@@ -122,8 +72,29 @@
           <portlet:param name="<%=PortalConstants.PAGENAME%>" value="<%=PortalConstants.PAGENAME_SOURCE_CODE_BUNDLE%>"/> \
         </portlet:renderURL>';
     }
+
+    var linkedProjectsTable;
     $(window).load(function () {
-        $("#LinkedProjectsInfo").treetable({expandable: true});
+        linkedProjectsTable = $("#LinkedProjectsInfo");
+        linkedProjectsTable.treetable({
```

Comment:
maybe move the code to some common `.jspf` file which gets imported in both places?